doc/releases.rst

```diff
@@ -295,34 +295,59 @@ Timeline
 Understanding the release cycle
 -------------------------------
 
-The development release cycle is two to four weeks long.  Each cycle
-freezes the master development branch and applies `integration and
-upgrade tests <https://github.com/ceph/ceph-qa-suite>`_ for the
-duration of one cycle before it is released and the next release's
-code is frozen for testing.  Once released, there is no effort to
-backport fixes; developer focus in on the next development release
-which is usually only a few weeks away.
-
-There are three to four stable releases a year.  Each stable release
-will receive a name (e.g., 'Jewel') and bug fix backports at least
-until the next stable release is out.
-
-Every other stable releases is a LTS (Long Term Stable) and will
-receive updates until two LTS are published. For instance Dumpling is
-retired when Hammer is published, Firefly is retired when Jewel is
-published etc. The rationale is that backports to a LTS (Firefly for
-instance) are expected to happen until the next LTS is published
-(Jewel is the LTS following Hammer), to fix bugs and possibly
-backport important features. After the next LTS is published,
-backports are still expected to fix bugs with a focus on whatever can
-prevent upgrades to the next LTS (in our example, fixes to Dumpling
-were published after Firefly was released and until Hammer was
-published, primarily to ensure Dumpling cluster can smoothly migrate
-to Firefly).
-
-* Long Term Stable : until the next two LTS are published
-* Stable release : until the next stable release is published
-* Development / testing release : no backports
+There is a new stable release cycle every nine (9) months.  Each
+stable release series will receive a name (e.g., 'Jewel') and a major
+release number (e.g., 10 for Jewel because 'J' is the 10th letter of
+the alphabet).  Each release has a version like 10.y.z.
+
```

Comment:
Maybe we should use "Mimic" examples in section as this is new policy which wasn't present when Jewel was current?